support/package-builder/prepare-build-root.sh

```diff
@@ -47,15 +47,17 @@ if mountpoint ${BUILDROOT}/dev	>/dev/null 2>&1; then umount ${BUILDROOT}/dev; fi
 #Untar tools build to buildroot
 #
 
-echo "Uncompressing the tools tar"
-
-tar -C ${BUILDROOT} -xvf $TOOLS_PATH/tools-build.tar >/dev/null
-# YUSTAS_FIXME: rpm still depends on /tools/lib/... Remove this dependency.
-ln -sf ${BUILDROOT}/tools /tools
+if [ "x$TOOLS_ARCHIVE" = "xminimal" ] ; then
+    echo "Uncompressing the tools tar (minimal)"
+    tar -C ${BUILDROOT} -xzf $TOOLS_PATH/tools.tar.gz >/dev/null
```

Comment:
tools-build.tar(700 MB) is being extracted only once to build RPMs of toolchains.
tools.tar.gz(only 8 MB) is being extracted for any others RPM's. It's small and extracts very fast. it's not necessary to 'cache' it.
PS: Idea to use absolute host paths (like /tools) is not good for parallelism.